libs/server/src/lib/utils/utils.ts

```diff
@@ -352,7 +355,9 @@ function renameProperty(obj: any, from: string, to: string) {
   delete obj[from];
 }
 
-export function toWorkspaceFormat(w: any): WorkspaceJsonConfiguration {
+export function toWorkspaceFormat(
```

Comment:
This function could reuse `toNewFormat` from @nrwl/tao

Reply:
These linked lines are probably not needed anymore since we use the NX helpers to read config

https://github.com/Cammisuli/nx-console/blob/use-nx-utils-for-configs/libs/server/src/lib/utils/utils.ts#L380-L382